themes/bootstrap3/js/common.js

```diff
@@ -655,7 +655,10 @@ function unwrapJQuery(node) {
 function setupJumpMenus(_container) {
   var container = _container || $('body');
   container.find('select.jumpMenu').on("change", function jumpMenu() {
-    $(this).parent('form').trigger("submit");
+    // Check if jumpMenu is still enabled (search.js may have disabled it):
```

Comment:
This is actually a workaround for jumpMenus, so if we can get rid of them, this can go. The purpose of this change is to be able to disable jumpMenu in case we reload the results asynchronously and don't want to jump. Since the js results init runs after this bind has been done, removing the jumpMenu class afterwards allows the action to be disabled. 

We may of course need another way to substitute the code that reloads the results if jumpMenu is replaced with something else that triggers a reload.